src/extras/core/PathPrototype.js

```diff
@@ -109,7 +109,7 @@ var PathPrototype = Object.assign( Object.create( CurvePath.prototype ), {
 			// if a previous curve is present, attempt to join
 			var firstPoint = curve.getPoint( 0 );
 
-			if ( ! firstPoint.equals( this.currentPoint ) ) {
+			if ( firstPoint.distanceToManhattan( this.currentPoint ) > 0.00001 ) {
```

Comment:
Out of curiosity, how do you know what the tolerance should be? What happens if the coordinates of the two points are very, very small -- or very, very large?

Reply:
It is just an approximation that should work in most cases. Do you think it would be better if the user applies a `tolerance` parameter to `. absellipse()`?

Reply:
Quite frankly, I'm not sure the best way to handle floating point problems like this.

I'm not a big fan of solutions that don't necessarily work, either, but I don't know what to tell you.

Reply:
Or maybe a configurable property like `.arcLengthDivisions`  of `Curve`?

Reply:
The problem is that the usage of pure `equals()` fails in this method in most cases. So any approach that uses a tolerance value should provide better results...

Reply:
Scaling the tolerance value based on the input vectors might be a way to go. But i don't feel this overhead is necessary here.

Reply:
Whatever you want to do to here is fine with me.

Reply:
Nevertheless, i will re-consider the approach. Closing the PR for now so it does not get accidentally merged.